concent_api/generate_requirements_lock.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+if [[ "$VIRTUAL_ENV" != "" ]]
+then
+    echo "--find-links https://builds.golem.network/simple/pyelliptic" > ${BASH_SOURCE%/*}/requirements.lock
+    pip freeze >> ${BASH_SOURCE%/*}/requirements.lock
+    echo "Done"
+else
+    echo "Script should be run within virtual environment"
```

Comment:
Please also add `exit 1` below. Now, when virtualenv is not running script ends with no error.